third_party/ienn/src/ie_compilation.cpp

```diff
@@ -316,25 +314,33 @@ int32_t Compilation::AddElementwise(const Operation& operation) {
   }
   try {
     const uint32_t output_index = operation.outputs[0];
+    std::shared_ptr<ngraph::Node> node;
     if (operation.type == ADD) {
       std::cout << "[IE] add ngraph Add"
-                << " for output operand index " << output_index;
-      auto add =
-          std::make_shared<op::v1::Add>(index_op_map_[operation.inputs[0]],
-                                        index_op_map_[operation.inputs[1]]);
-      index_op_map_[output_index] = add;
+                 << " for output operand index " << output_index;
+      node = std::make_shared<op::v1::Add>(index_op_map_[operation.inputs[0]],
+                                           index_op_map_[operation.inputs[1]]);
+      // index_op_map_[output_index] = add;
     } else if (operation.type == MUL) {
       std::cout << "[IE] add ngraph Multiply";
-      auto mul = std::make_shared<op::v1::Multiply>(
+      node = std::make_shared<op::v1::Multiply>(
           index_op_map_[operation.inputs[0]],
           index_op_map_[operation.inputs[1]]);
-      index_op_map_[output_index] = mul;
+      // index_op_map_[output_index] = mul;
       std::cout << "[IE] succeed to add ngraph Multiply "
                 << " for output operand index " << output_index;
     } else {
       std::cout << "Operation type " << operation.type << " is not supported";
       return error_t::BAD_DATA;
     }
+    if (params.fuse_code != fuse_t::FUSED_NONE) {
+      result = AddActivationByFusedCode(params.fuse_code, node, node);
+      if (result != error_t::NOT_ERROR) {
+        return result;
+      }
+    }
+    index_op_map_[output_index] = node;
```

Comment:
move this code to `AddActivationByFusedCode` so that the third parameter is unnecessary.